airbyte-cdk/python/unit_tests/sources/message/test_repository.py

```diff
@@ -74,33 +74,28 @@ def test_given_message_is_consumed_when_consume_queue_then_remove_message_from_q
         second_message_generator = repo.consume_queue()
         assert list(second_message_generator) == [second_message]
 
-    def test_given_message_is_not_control_nor_log_message_when_emit_message_then_raise_error(self):
-        repo = InMemoryMessageRepository()
-        with pytest.raises(ValueError):
-            repo.emit_message(AirbyteMessage(type=Type.STATE, state=AirbyteStateMessage(data={"state": "state value"})))
-
     def test_given_log_level_is_severe_enough_when_log_message_then_allow_message_to_be_consumed(self):
         repo = InMemoryMessageRepository(Level.DEBUG)
-        repo.log_message(Level.INFO, lambda: "this is a log message")
+        repo.log_message(Level.INFO, lambda: {"message": "this is a log message"})
```

Comment:
This change is not because of a change of behavior. It is just fixing typing as it was not aligned with the method signature